packages/schematics/update/update/index.ts

```diff
@@ -780,10 +780,11 @@ export default function(options: UpdateSchema): Rule {
     const packages = _buildPackageList(options, allDependencies, logger);
     const usingYarn = options.packageManager === 'yarn';
 
-    return observableFrom([...allDependencies.keys()]).pipe(
+    return observableFrom(allDependencies).pipe(
       // Grab all package.json from the npm repository. This requires a lot of HTTP calls so we
       // try to parallelize as many as possible.
-      mergeMap(depName => getNpmPackageJson(depName, options.registry, logger, usingYarn)),
+      mergeMap(([depName, version]) =>
+        getNpmPackageJson(`${depName}@${version}`, options.registry, logger, usingYarn)),
```

Comment:
Was not certain if to add an addition param or not.

Reply:
That function is intended to pull all version information from the registry not just a particular version.  A more complex change is needed to the full update algorithm to account for package data that does not contain full version history.